<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QMainWindow" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>483</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLineEdit" name="commandWriter">
       <property name="placeholderText">
        <string>Entrer une commande</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QTextBrowser" name="msgLog">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="data.qrc">
         <normaloff>:/images/icons8_usb_connector_32.png</normaloff>:/images/icons8_usb_connector_32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>29</width>
         <height>33</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="data.qrc">
         <normaloff>:/images/icons8_synchronize_32.png</normaloff>:/images/icons8_synchronize_32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>29</width>
         <height>28</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comportsBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFindComPorts">
   <property name="text">
    <string>FindComPorts</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
